src/main/java/seedu/address/commons/util/QrUtil.java

```diff
@@ -57,9 +57,7 @@ public static void generateQr(String data) {
             BufferedImage bufferedImage = SwingFXUtils.fromFXImage(writableImage, null);
             ImageIO.write(bufferedImage, "png", file);
             //@@author
-        } catch (WriterException e) {
-            e.printStackTrace();
-        } catch (IOException e) {
+        } catch (WriterException | IOException e) {
             e.printStackTrace();
```

Comment:
Partly my fault but we will need to handle the exceptions outside of this function in the future. Because if it fails, it will fail silently and we will not be able to handle it in our command.